src/main/resources/model/authority.json

```diff
@@ -164,22 +164,24 @@
       "type": "object",
       "properties": {
         "createdDate": {
-          "index": "source"
+          "searchTypes": "filter",
+          "index": "date"
         },
         "createdByUserId": {
-          "index": "source"
+          "index": "keyword"
```

Comment:
changing the mapping for the fields:
- createdByUserId
- createdByUsername
- updatedByUserId
- updatedByUsername 
is not the part of the story, can you revert it? (it's possible that this field would never be used for searching purposes